src/Components/App/App.js

```diff
@@ -25,6 +25,23 @@ function App() {
     .catch(err => console.log(`There has been an error: ${err}`))
   }, [dispatch]);
 
+  useEffect(() => {
+    getFriends(1)
+      .then(data => {
+        console.log('friends', data.data.friends);
+      })
+      .catch(err => console.log(`There has been an error: ${err}`))
+  }, []);
+
+  useEffect(() => {
+    getGroups()
+    .then(data => {
+      console.log(data)
+      dispatch(updateGroups(data));
+    })
+    .catch(err => console.log(`There has been an error: ${err}`))
+  }, [dispatch]);
+  
```

Comment:
Need to change this PR, as well - remove the `useEffect` for `getFriends`, and move the `useEffect` for `getGroups` to the GroupPage component